ec2system/ec2machine.go

```diff
@@ -268,7 +267,10 @@ func (s *System) Init(b *bigmachine.B) error {
 		s.Diskspace = 200
 	}
 	if s.Binary == "" {
-		s.Binary = "https://grail-public-bin.s3-us-west-2.amazonaws.com/linux/amd64/ec2boot0.4"
+		s.Binary = defaultEc2Boot
+	} else if s.Binary != defaultEc2Boot && strings.HasPrefix(s.Binary, defaultEc2BootPrefix) {
```

Comment:
```suggestion
	} else if strings.HasPrefix(s.Binary, defaultEc2BootPrefix) {
```
Same effect?

Reply:
no: I don't want the log message to be printed when `s.Binary == default...`